build.gradle

```diff
@@ -109,6 +109,10 @@ allprojects {
         }
         json {
             target "src/**/*.json"
+            // schemaVersion must be at the top or else Fabric will complain
+            targetExclude("src/main/resources/fabric.mod.json")
+            // spotless replaces the Unicode escapes with the actual characters and rearranges the order, worsening readability.
```

Comment:
The order should be alphabetical, even if it worsen readability. That's the only way to get consistency. I accept that you need to exclude it from spotless if it removes the escape sequence, but please let spotless order the file once first.